lib/docker_compose/stack.rb

```diff
@@ -6,7 +6,7 @@ class Stack
     def initialize(stack, using:)
       @architecture = using
       @stack = stack.with_indifferent_access
-      @data = { version: architecture.version, volumes: architecture.volumes }.with_indifferent_access
+      @data = { version: architecture.version }.with_indifferent_access
```

Comment:
What does this change do?

Reply:
We used to have the `volumes` field in the `docker-compose-full.yml` file, which needed to be in the `docker-compose.yml` file that is generated. Now that the `volumes` field has been removed from the `docker-compose-full.yml` file, not removing it from the above code would result in an empty `volumes` key in the `docker-compose.yml`, which `docker-compose` will complain about.